guides/v2.1/cloud/project/project-routes-more-redir.md

```diff
@@ -34,37 +36,35 @@ http://{default}/:
             "/regexp/(.*)/matching": { "to": "http://example.com/$1", regexp: true }
 ```
 
-This format is more rich and works with any type of route, including routes served directly by the application.
+You can use partial redirects with any type of route, including routes served directly by the application.
 
 Two keys are available under `redirects`:
 
--  `expires`: Optional, the duration that the redirect is cached.
+- `expires`—Optional, the duration that the redirect is cached.
```

Comment:
Inherited, but both of these descriptions lack in quality.

Reply:
This whole section is not great. Will clean up on next go around.

Reply:
Revised the Partial-route redirects section entirely to provide better description for the `paths` key and associated properties, and place examples in a separate section.  Please check.